xarray/plot/plot.py

```diff
@@ -102,6 +111,26 @@ def _line_facetgrid(darray, row=None, col=None, hue=None,
     return g.map_dataarray_line(hue=hue, **kwargs)
 
 
+def _convert_cftime_data(values):
```

Comment:
I would move these to `utils.py` because they might come in handy for a Dataset plotting API.

Reply:
Do you still think that this could be of use? Otherwise Ill go ahead and remove it.